examples/article-access-laterpay.amp.html

```diff
@@ -9,8 +9,11 @@
   <script id="amp-access" type="application/json">
     {
       "vendor": "laterpay",
-      "laterpay": {},
-      "authorizationFallbackResponse": {"error": true}
+      "authorizationFallbackResponse": {"error": true},
+      "noPingback": true,
```

Comment:
Why `noPingback = true`? I don't think you want this. The access system will delegate this call to the `pingback` method in your service and you can process it as you like, but it's better not advise it in the sample docs.